Documentation/ReleaseNotes/5.0rc02.md

```diff
@@ -0,0 +1,479 @@
+
+The Python package synchronization infrastructure for static global variables was refactored to support more variables and permit future additions to the singleton. This will prevent the need to rebuild remote module binary Python packages following future releases of ITK 5. However, current remote module Python packages that use ITK 5 should be rebuilt and republished for ITK 5.0 RC 2.
+
+For an overview of ITK 5's transition to modern C++, performance-related changes, the new, Pythonic API, the project's migration to GitHub, and Mesh improvements, see the [ITK 5 Alpha 1: Modern C++](https://discourse.itk.org/t/itk-5-0-alpha-1-modern-c/843/7), [ITK 5 Alpha 2: Performance](https://discourse.itk.org/t/itk-5-0-alpha-2-performance/959), [ITK 5 Beta 1: Pythonic Interface](https://discourse.itk.org/t/itk-5-0-beta-1-pythonic-interface/1271), [ITK 5 Beta 3: GitHub](https://discourse.itk.org/t/itk-5-0-beta-3-github/1504), [ITK 5 Release Candidate 1: Meshes](https://discourse.itk.org/t/itk-5-0-release-candidate-1-meshes/1576/3) release announcements.
+
+New Remote Modules
+------------------
+
+- [`ITKIOScanco`](https://github.com/KitwareMedical/ITKIOScanco): read and write Scanco microCT *.isq* files.
+- [`ITKTotalVariation`](https://github.com/InsightSoftwareConsortium/ITKTotalVariation): total variation image denoising.
+- [`ITKRTK`](https://github.com/SimonRit/RTK): the Reconstruction Toolkit provides high performance implementations of advanced tomographic image reconstruction algorithms.
+- [`ITKThickness3D`](https://github.com/InsightSoftwareConsortium/ITKThickness3D): compute the skeleton and thickness transform from 3D images.
+
+Performance Improvements
+------------------------
+
+- Add MetaDataDictionary move support.
+- MetaDataDictionary uses copy-on-write.
+- `itk::Barrier` is now deprecated: replace with the new multi-threading functions.
+- Range-based for loops supported in `itk::FixedArray`.
+
```

Comment:
I would suggest adding to this section:
`Major performance improvement TimeProbe, ResourceProbe constructor (issue #350)`

Reply:
Thanks, @N-Dekker -- added.